pkg/pod/script.go

```diff
@@ -68,6 +70,25 @@ func convertScripts(shellImage string, steps []v1alpha1.Step) (*corev1.Container
 			continue
 		}
 
+		// Check for a shebang, and add a default if it's not set.
+		// The shebang must be the first non-empty line.
+		lines := strings.Split(s.Script, "\n")
+		hasShebang := false
+		for _, l := range lines {
+			if strings.TrimSpace(l) == "" {
```

Comment:
Does TrimSpace ignore blank lines? I'd love to be able to avoid splitting all the lines just to find the first non-empty one.